bot/code_coverage_bot/phabricator.py

```diff
@@ -109,6 +145,12 @@ def generate(self, report, changesets):
 
                 # For each file...
                 for path in changeset["files"]:
+
+                    # Skip third party files
```

Comment:
I wouldn't skip them here, if we have a third-party file on a Phabricator revision, we still want to upload its coverage (but we don't want to warn about it).